.gitlab-ci.yml

```diff
@@ -59,6 +60,7 @@ generate_sbom:
     - .run_sbom_generation
   stage: sbom
   variables:
+    ARTIFACTORY_UPLOAD_REPO: 1
```

Comment:
what is this for?

Reply:
This change was introduced by Mike. To ensure request to upload to artifactory is honored by setting env var ARTIFACTORY_UPLOAD_REPO to 1
<img width="1671" alt="Screenshot 2023-03-27 at 12 48 41 PM" src="https://user-images.githubusercontent.com/104522232/227869358-c86843dc-c345-4c7a-800b-9e8ad20dabee.png">